src/components/textarea/textarea.scss

```diff
@@ -98,7 +98,7 @@ label {
     }
 
     &[disabled] {
-      color: var(--color-smoke-50);
+      color: $input-disabled-color;
```

Comment:
We ultimately have to move to specific colors in colors.scss. Can we maybe use the CSS vars? Maybe a darker shade?